ui/desktop/app/routes/scopes/scope/projects.js

```diff
@@ -49,4 +54,46 @@ export default class ScopesScopeProjectsRoute extends Route {
     );
     return projects;
   }
+
+  /**
+   * Poll for new sessions with credentials. Sends a notification for each new session that has a credential.
+   */
+  @runEvery(2000)
+  async poller() {
+    const sessions =
+      await this.clientAgentSessions.getNewSessionsWithCredentials();
+
+    sessions.forEach((session) => {
+      new window.Notification(
+        this.intl.t('notifications.connected-to-target.title', {
+          target: session.alias,
```

Comment:
Why are we passing this target info? Do we need this for the notification?

Reply:
This is so we have a name for the what we're connecting to
![image](https://github.com/hashicorp/boundary-ui/assets/5783847/6649cbb3-d2fa-4500-9c47-e358c8c4a794)

Reply:
My bad. I didn't realize that was the session's name :D